docs/reference/commandline/cp.md

```diff
@@ -16,7 +16,8 @@ weight=1
 
     Copy files/folders between a container and the local filesystem
 
-      --help=false        Print usage
+      -L, --follow-link=false    Always follow symbol link in source
```

Comment:
>@WeiZhang555 not your fault but looking at the cp command it looks like the basic structure >of the command description is flawed. If we restructure the command as a whole to use >CONTAINER:SRC/DEST_PATH I think it is easier to follow the content.
>
>Would you be willing to do this? If not, I can carry and ask you to do a technical review.

I'd like to help, always, so please let me do it if you have a good idea :smile: . 
But first of all, I couldn't quite get it. Do you mean we need a UX change? Can you explain a little bit more? thank you!